levelCreator/states/create.js

```diff
@@ -32,8 +34,22 @@ function initCreateState() {
 
   state.create = function() {
     const game = window.game;
-    gusSpawn = game.add.sprite(-16, -16, 'Gus');
-    game.stage.setBackgroundColor(COLORS.DEFAULT_SKY)
+    gusSpawn = game.add.sprite(0, 0, 'Gus');
+    game.stage.setBackgroundColor(COLORS.DEFAULT_SKY);
+
+    game.dolly = new Dolly( game.camera );
+    game.dolly.targetPos = new Phaser.Point( 0, 0 );
+
+    // Set Keyboard input
+    upKey = game.input.keyboard.addKey(Phaser.Keyboard.W);
+    downKey = game.input.keyboard.addKey(Phaser.Keyboard.S);
+    leftKey = game.input.keyboard.addKey(Phaser.Keyboard.A);
+    rightKey = game.input.keyboard.addKey(Phaser.Keyboard.D);
```

Comment:
I think we could make these into one object that stores each of the keys instead of 4 different vars